torch/csrc/distributed/rpc/request_callback_impl.cpp

```diff
@@ -57,6 +61,17 @@ std::unique_ptr<RpcCommandBase> deserializePythonRpcCommandReference(
       }
       return nullptr;
     }
+    case MessageType::RUN_WITH_PROFILING_REQ: {
```

Comment:
It's my bad to introducing this ugly unwrap. Added #39351 to track, and will try to solve this after v1.6 branch cut.